qemu/tests/cfg/pci_hotplug.cfg

```diff
@@ -117,6 +117,8 @@
             only hotplug_block
             pci_model = virtio
             match_string = "Virtio block device"
+            Windows:
+                match_string = "VirtIO SCSI Disk"
```

Comment:
Yes, but this is what our device register in windows system. Confirmed with feature owner.